src/parser.ts

```diff
@@ -29,7 +29,7 @@ function handleAttrs(token: Token, type: string) {
       return { ordered: token.type.startsWith('ordered') };
     case 'link': {
       const attrs = Object.fromEntries(token.attrs);
-      return { href: attrs.href };
+      return attrs?.title ? { href: attrs.href, title: attrs.title } : { href: attrs.href };
```

Comment:
`Object.fromEntries` always returns an object so you don't need `?.`.

But also if you check the code around, no need to have a ternary at all, `undefined` value is just fine.

```suggestion
      return { href: attrs.href, title: attrs.title };
```

Reply:
You are right since the `{ href: attrs.href }` has undefined `href` too.